src/profiles/y_profile_admin_classes.prog.abap

```diff
@@ -677,7 +682,12 @@ CLASS lcl_util IMPLEMENTATION.
   METHOD refresh_checks.
     CHECK checks_tree IS BOUND.
     TRY.
-        checks_tree->list_control( )->set_table( profile_manager->select_checks( get_selected_profile( )-profile ) ).
+        DATA(checks) = profile_manager->select_checks( get_selected_profile( )-profile ).
+        LOOP AT checks ASSIGNING FIELD-SYMBOL(<check>).
+          <check>-ignore_pseudo_comments = switch_bool( <check>-ignore_pseudo_comments ).
+        ENDLOOP.
```

Comment:
Or, we can move it to a new method and describe this workaround in the method's body itself.